### PR TITLE
avoid warnings from pytest-httpbin

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
see https://github.com/kevin1024/pytest-httpbin/pull/71

**Comment by tomchristie** [comment]
Oh neat. I guess we wait until there's a PyPI version of `pytest-httpbin` with this. (?)

**Comment by graingert** [comment]
> Oh neat. I guess we wait until there's a PyPI version of `pytest-httpbin` with this. (?)

I made a pypi release

File: requirements.txt [hunk 1]
```diff
@@ -23,6 +23,7 @@ flake8==4.0.1
 isort==5.10.1
 mypy==0.931
 pytest==7.0.1
-pytest-httpbin==1.0.1
+pytest-httpbin==2.0.0rc1; python_version >= '3.7'
+pytest-httpbin==1.0.2; python_version < '3.7'
```
Comment (graingert) [inline requirements.txt:27]: https://github.com/encode/httpcore/issues/515

**Comment by tomchristie** [comment]
Tests all clear.
We've dropped 3.6 now, so this can be updated accordingly.

Thanks @graingert!

File: requirements.txt [hunk 2]
```diff
@@ -24,7 +24,8 @@ flake8==3.9.2  # See: https://github.com/PyCQA/flake8/pull/1438
...
```
Comment (graingert) [inline requirements.txt:28]:
```suggestion
```

File: requirements.txt [hunk 3]
```diff
@@ -24,7 +24,8 @@ flake8==3.9.2  # See: https://github.com/PyCQA/flake8/pull/1438
 isort==5.10.1
 mypy==0.931
 pytest==7.0.1
-pytest-httpbin==1.0.1
+pytest-httpbin==2.0.0rc1; python_version >= '3.7'
```
Comment (graingert) [inline requirements.txt:27]:
```suggestion
pytest-httpbin==2.0.0rc1
```

**Review by Kludex — APPROVED** [review]

**Pull request merged**